spec/feature/user_system_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :system do
+  describe 'displays all users' do
+    it 'displays all users' do
```

Comment:
- [ ] Once more let's kindly make sure all our tests are passing.

![Screenshot from 2023-03-22 13-17-00](https://user-images.githubusercontent.com/63087491/226902696-1081dc7d-3fc9-470e-ab5b-082ada3dc2af.png)

- [Optional] You're using two `describe` blocks with a single test which I think it's not a good practice. Also, it is difficult to understand the output of the string arguments passed into the `describe` and `it` blocks. Kindly check out the output on the console below and fix it.

![Screenshot from 2023-03-22 13-19-07](https://user-images.githubusercontent.com/63087491/226904286-d2940432-533e-48db-a920-1ca34e0ba69b.png)